src/Telnyx.net/Entities/Messaging/Messaging_Sender_Ids/MessagingSenderId.cs

```diff
@@ -4,6 +4,54 @@ namespace Telnyx
     using System.Runtime.Serialization;
     using Newtonsoft.Json;
 
+    public class MessagingFrom
```

Comment:
I believe this should me MessagingTo according to these docs https://developers.telnyx.com/docs/api/v2/messaging/Messages

To and From should be switched as the From field is a string

Reply:
Yeah... the docs are wrong apparently or this needs to be a Union type.